### DISCUSSION TITLE
CouchDB Docs getting Corrupted

### DISCUSSION THREAD

**Discussion by Sourabh-Zutti** [apache/couchdb#3714] [Q&A] [answered]
What might be the reason to get data corrupted
I am using CouchDB version: 1.6.1
In our project, we make all kinds of transactions with CouchDB like add, delete, edit.
I am having many docs, for which I have written views, but it gives an error in an alert box as Error: file_corruption
The view consists of thousands of doc, to debug it gets difficult.
There is no disc space problem.
Is there any way to get the list of all corrupted file lists and what might be reasons to get data corrupt?

Thank you in advance

**Comment by nickva** [comment, marked as answer]
Could be another server running on the same data volume, using NTFS (or other non POSIX file systems) by chance?

1.6.1 is quite a bit outdated and unsupported, consider upgrading to the latest 3.x version
